src/group-configurations/data/api.js

```diff
@@ -4,7 +4,14 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 const API_PATH_PATTERN = 'group_configurations';
 const getStudioBaseUrl = () => getConfig().STUDIO_BASE_URL;
 
-export const getGroupConfigurationsApiUrl = (courseId) => `${getStudioBaseUrl()}/api/contentstore/v1/${API_PATH_PATTERN}/${courseId}`;
+export const getContentStoreApiUrl = (courseId) => `${getStudioBaseUrl()}/api/contentstore/v1/${API_PATH_PATTERN}/${courseId}`;
+export const getLegacyApiUrl = (courseId, parentGroupId, groupId) => {
+  const url = `${getStudioBaseUrl()}/${API_PATH_PATTERN}/${courseId}${
+    parentGroupId ? `/${parentGroupId}` : ''
+  }${groupId ? `/${groupId}` : ''}`;
+
+  return url;
+};
```

Comment:
[nit]: Maybe it will look more readable this way

```suggestion
export const getLegacyApiUrl = (courseId, parentGroupId, groupId) => {
  const BASE_URL = `${getStudioBaseUrl()}/${API_PATH_PATTERN}/${courseId}`;
  const PARENT_GROUP_PATH = `${BASE_URL}${parentGroupId ? `/${parentGroupId}` : ''}`;
  const GROUP_PATH = `${groupId ? `/${groupId}` : ''}`;

  return `${BASE_URL}${PARENT_GROUP_PATH}${GROUP_PATH}`;
};
};
```

Reply:
agree with you, it looks more readable, changed